test/test_pmm.py

```diff
@@ -111,6 +111,21 @@ def setUp(self):
             minimum_spread=-1,
         )
 
+        self.order_override_strategy = PureMarketMakingStrategy(
+            self.market_info,
+            bid_spread=Decimal("0.01"),
+            ask_spread=Decimal("0.01"),
+            order_amount=Decimal("1"),
+            order_refresh_time=5.0,
+            filled_order_delay=5.0,
+            order_refresh_tolerance_pct=-1,
+            order_levels=3,
+            order_level_spread=Decimal("0.01"),
+            order_level_amount=Decimal("1"),
+            minimum_spread=-1,
+            order_override={"override_order_one": ["buy", "1", "1"], "override_order_two": ["sell", "1", "1"]},
```

Comment:
Please add a test to check that it will clearly override the spread, order_amount and order levels, e.g. ["buy", "0.5", "0.7"],  ["buy", "1.3", "1.1"], ["sell", "1", "1"]

Reply:
I added the above 3 examples to unit test.